packages/core/src/browser/core-preferences.ts

```diff
@@ -136,6 +136,20 @@ export const corePreferenceSchema: PreferenceSchema = {
             default: isOSX ? 1500 : 500,
             description: nls.localizeByDefault('Controls the delay in milliseconds after which the hover is shown.')
         },
+        'workbench.sash.hoverDelay': {
+            type: 'number',
+            default: 0,
```

Comment:
Looking at the [VSCode codebase](https://github.com/microsoft/vscode/blob/2085b19a51fcab47f4991f51ae8f9eda7923c1d7/src/vs/workbench/contrib/sash/browser/sash.contribution.ts#L35), I noticed that they use a delay of `300`. I think we should align to that.